playground/backend/containers/java/Dockerfile

```diff
@@ -69,5 +69,11 @@ RUN wget https://repo1.maven.org/maven2/org/hamcrest/hamcrest-all/$HAMCREST_VERS
 # Install JUnit
 RUN wget https://repo1.maven.org/maven2/junit/junit/$JUNIT_VERSION/junit-$JUNIT_VERSION.jar &&\
     mv junit-$JUNIT_VERSION.jar /opt/apache/beam/jars/junit.jar
+# Install Java Katas Utils
+COPY katas /go/src/katas
+WORKDIR /go/src/katas
```

Comment:
do we need to switch workdir for future commands? or we can add cd /go/src/katas && to line 75?

Reply:
Fair enough, changed